src/playwright/playwright.config.ts

```diff
@@ -17,7 +17,6 @@ export default defineConfig({
       threshold: 0.03,
     },
   },
-  failOnFlakyTests: Boolean(process.env.CI),
```

Comment:
A flaky test is what you'd expect: it's one that fails initially but succeeds on a retry. I removed this because I removed the `retries`. More on that below.